### PR TITLE
A0-2250: change clipboard clear flow

### PR DESCRIPTION
**Pull request by youPickItUp**

### CODE REVIEW

File: packages/extension-ui/src/util/clearClipboard.ts [hunk 1]
```diff
@@ -12,3 +11,8 @@ export default () => {
   form.blur();
   document.body.removeChild(form);
 };
+
+export default () => {
+  navigator.clipboard.writeText(' ').catch(() => null);
+  clearClipboardFallback();
```
Comment (jalooc) [inline packages/extension-ui/src/util/clearClipboard.ts:17]: Maybe rather:
```suggestion
  navigator.clipboard.writeText(' ').catch(() => clearClipboardFallback());
```
?

Comment (youPickItUp) [reply packages/extension-ui/src/util/clearClipboard.ts:17]: It makes a lot of sense, I was just freaking out that if I use async and wait for result my narrow window of opportunity to interact with clipboard after user interaction will pass and everything will be lost :scream:

I'll change it :+1:

File: packages/extension-ui/src/components/AccountCreationSuccess.tsx [hunk 2]
```diff
@@ -1,51 +1,48 @@
...
-    // Clearing the clipboard from the background thread to have it done even after the popup is closed
-    clearClipboard(CLIPBOARD_CLEANUP_TIMEOUT);
-
-    const timeout = setTimeout(() => {
-        window.close();
-    }, CLIPBOARD_CLEANUP_TIMEOUT);
-
-    return () => clearTimeout(timeout);
+    // clear clipboard when user closes current window
+    window.addEventListener('beforeunload', clearClipboard);
```
[the remark this answers is not in the document]
Comment (youPickItUp) [reply packages/extension-ui/src/components/AccountCreationSuccess.tsx:21]: I'm a bit afraid that unmount will clear my listener before it's called... not sure if that a possibility :sweat_smile:

[the remark this answers is not in the document]
Comment (youPickItUp) [reply packages/extension-ui/src/components/AccountCreationSuccess.tsx:21]: I stand corrected, it's definitely **not** a  possibility, I'll change that

**Pull request merged**